### PR TITLE
Replit-ify Personal Website Workshop

### PR DESCRIPTION
**Pull request by yevbar**
I don't expect this to be a quick PR however the README is in a complete state and in need of edits

### CODE REVIEW

File: workshops/personal_website/README.md [hunk 1]
```diff
@@ -1,27 +1,25 @@
 ---
 name: Personal Website
-description: Your first website from scratch
-author: Hack Club staff
+description: Making your first website from scratch
+author: Hack Club staff + Yev
```
Comment (zanedb) [inline workshops/personal_website/README.md]: Yev you can't do this

Comment (yevbar) [reply workshops/personal_website/README.md]: Yes I can

**Comment by yevbar** [comment]
@MaxWofford Hmm, noted!

**Comment by MaxWofford** [comment]
This is going to take a bunch of time to review. I'm going to merge for the time being, then fix any issues I find.

**Pull request merged**